### ISSUE TITLE
Re-add `security` label to security-fix Dependabot (v2) PRs as in Dependabot v1

### ISSUE THREAD

**Issue by JacobEvelyn** [isaacs/github#1839]
**TL;DR:** Dependabot's security PRs should have a `security` label (or a way to configure this—for example a `security-labels` config option).

---

### More context

Our organization is committed to using Dependabot to keep all of our dependencies up to date, and this responsibility is shared amongst all our engineers. To remind us and track trends over time, we built a small Slack bot that uses the GitHub API and sends a message like this once per week:

<img width="452" alt="Screen Shot 2020-08-11 at 2 32 32 PM" src="https://user-images.githubusercontent.com/1114569/89935130-a9222a80-dbdf-11ea-8761-9e6defce3f3e.png">

At Dependabot's official recommendation, we use Dependabot v1, because v2 does not (yet?) support some features we use extensively, like auto-merging and private git dependencies:

<img width="458" alt="Screen Shot 2020-08-11 at 2 39 17 PM" src="https://user-images.githubusercontent.com/1114569/89935791-bdb2f280-dbe0-11ea-8a1f-5b1b586c8dea.png">

### Previous behavior

Previously, security-fix PRs were created by Dependabot v1 (the `dependabot-preview[bot]` user), and these conveniently all had the `security` label in addition to the `dependencies` default label. That allowed our Slack bot to include links (the `30` and `1 security` in the screenshot) to GitHub search results with the following queries, respectively:
    - `is:open is:pr org:our-org label:dependencies`
    - `is:open is:pr org:our-org label:security`

### Current behavior

Recently, all security-fix PRs have been created by Dependabot v2 (the `dependabot[bot]` user), and these no longer have the `security` label, meaning **there is no way** for us to easily see all of the security-fix PRs across all of our repos, or have an engineer quickly find one to review and merge.

### Our request

If Dependabot could **start adding the `security` label to its security PRs** it would save us a ton of time and effort, and prevent us from missing important PRs in infrequently-updated repos. If there are backwards-compatibility concerns you could even make this configurable via a `security-labels` config option like [the `labels` one](https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#labels).
